.github/cdk/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@pennlabs/kraken": "^0.8.6",
     "cdkactions": "^0.2.3",
-    "constructs": "^3.3.263"
+    "constructs": "^3.2.109"
```

Comment:
This is the version used by `cdkactions`. Using a `3.3.x` version would cause some products (e.g. penn-clubs to break)

Reply:
```
main.ts:6:26 - error TS2345: Argument of type 'App' is not assignable to parameter of type 'Construct'.
  Property 'onValidate' is protected but type 'Construct' is not a class derived from 'Construct'.
  ```